app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -58,6 +58,11 @@ def track_logout_event
       oidc: false,
       saml_request_valid: sp_initiated ? valid_saml_request? : true,
     )
+    irs_attempts_api_tracker.logout_initiated(
+      user_uuid: "",
```

Comment:
ok so I think that `user_uuid` should be implicitly added to every event, similar to how it is added for all Analytics events by default:

https://github.com/18F/identity-idp/blob/main/app/controllers/application_controller.rb#L66-L87

Reply:
but in terms of literally having access to the data here, that would be:
```suggestion
      user_uuid: current_user.uuid,
```

Reply:
I agree that setting this up in the Tracker constructor is a good idea, but I'm hoping to implement this in [LG-7058](https://cm-jira.usa.gov/browse/LG-7058).

We just talked about the `current_user` issue. For posterity, it appeared to be unavailable, but the problem was actually with test setup. Calling `stub_sign_in(user)` gets us what we need.

Reply:
I have a crude PR at https://github.com/18F/identity-idp/pull/6664 that will implement this.